### PR TITLE
fix: don't show a continuous histogram if there is only one value

### PR DESCRIPTION
**Pull request by Sparkier**

### CODE REVIEW

File: backend/zeno_backend/processing/histogram_processing.py [hunk 1]
```diff
@@ -124,8 +124,13 @@ async def histogram_metric_and_count(
     else:
         calculate_histograms = False
 
+    # if end = start, can't show a bar, remove bucket (only happens if signular value)
```
Comment (cabreraalex) [inline backend/zeno_backend/processing/histogram_processing.py:127]: sp, singular

Comment (cabreraalex) [reply backend/zeno_backend/processing/histogram_processing.py:127]: also ==

Comment (cabreraalex) [reply backend/zeno_backend/processing/histogram_processing.py:127]: Does this not show the feature in the histogram then? is that what we want to happen? or do we want to just show the singular bar so you know what the single value is for everything?

**Review by cabreraalex — CHANGES_REQUESTED** [review]

File: backend/zeno_backend/processing/histogram_processing.py (hunk 1 above)
Comment (Sparkier) [reply backend/zeno_backend/processing/histogram_processing.py:127]: Previously the histogram was completely empty for some reason so I removed it.

Don't know if we want it single value histogram is not useful really but you see that there is a column... could go either way.

**Review by cabreraalex — APPROVED** [review]

**Pull request merged**